.github/workflows/knative-go-test.yaml

```diff
@@ -14,6 +14,6 @@ on:
 
 jobs:
   test:
-    uses: knative/actions/.github/workflows/reusable-go-test.yaml@main
+    uses: cardil/knative-actions/.github/workflows/reusable-go-test.yaml@feature/unit-test-each-go-module
```

Comment:
/hold

Restore when https://github.com/knative/actions/pull/223 is merged

Reply:
Approved, the PR should be merged shortly.